### PR TITLE
fixes set 1

### LINKED ISSUE #149: Minor fixes
- timezone drop down not pulling up results in City add/edit page
- Unable to attach city to zip, or vice versa, city/zip dropdowns don't have results
- Attach city/zip 'primary' field error - incorrect value true/false should 1/0
- Add fields to country nova resource
- Unique validation on state, country nova resource fields
- Country calling code index, search by country
- fix city fieldsforindex
- cannot add/edit country callingcode, missing root and suffix fields
- fix phone area sort by state
- add missing fields to zip nova resource, including parent_zip_code relationship
- search by state in zips index
- search by country in states index
- remove creator/updater traits from Timezones model (TBD - alternatively should we add them to the migration and nova resource)

### PR DESCRIPTION
**Pull request by arwaawan3**
Closes #149 #143

### CODE REVIEW

**Comment by arwaawan3** [comment]
This is still work in progress. I will continue to test remaining nova resources tomorrow.

**Comment by arwaawan3** [comment]
I am going to test these resources more but I think we can review and merge the updates we have so far.

**Comment by arwaawan3** [comment]
More fixes:
- state abbreviation max 2 characters
- cities: make title and slug required fields
- phone areas index: able to search by state
- country calling code: unique constraint validation on code, country combination
- zips: Only allow numbers as acceptable characters. 'unique:zips,code'

File: src/Nova/Phone.php [hunk 1]
```diff
@@ -51,9 +51,11 @@ public function fields(Request $request)
             Text::make('Full Number', 'full_number', function () {
                 if (! empty($this->formatted_number)) {
                     return $this->formatted_number;
+                } else {
+                    return $this->full_number;
```
Comment (joshtorres) [inline src/Nova/Phone.php:55]: You could do here the same as line 35

File: src/Models/Timezone.php [hunk 2]
```diff
@@ -7,15 +7,11 @@
 use Tipoff\Addresses\Transformers\TimezoneTransformer;
 use Tipoff\Support\Contracts\Addresses\TimezoneInterface;
 use Tipoff\Support\Models\BaseModel;
-use Tipoff\Support\Traits\HasCreator;
 use Tipoff\Support\Traits\HasPackageFactory;
-use Tipoff\Support\Traits\HasUpdater;
 
 class Timezone extends BaseModel implements TimezoneInterface
 {
     use HasPackageFactory;
```
Comment (joshtorres) [inline src/Models/Timezone.php:14]: I'm assuming you double checked that we don't need these fields

**Review by joshtorres — APPROVED** [review]

File: src/Models/Timezone.php (hunk 2 above)
Comment (arwaawan3) [reply src/Models/Timezone.php:14]: Yes, the timezone table doesn't have created_at, updated_at or creator_id, updater_id. The alternative is to add those to the table and revert these changes.

**Pull request merged**